src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,17 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import { User } from '../../types/User';
+
+type Propss = {
+  title: string;
+  person: User ;
```

Comment:
```suggestion
  person: User | null;
```
about that: https://github.com/mate-academy/react_static-list-of-todos/pull/1519#discussion_r1084937068
and check user here